tests/unit/test-cmd/test.build.js

```diff
@@ -64,301 +56,271 @@ describe('build', () => {
+        messageFile: messageFileName,
+        manifestData: manifestWithRepeatingPattern,
+      });
+      assert.match(result, /example extension example extension/);
+    }
+  ));
 
-  it('checks locale file for malformed json', () => {
-    return withTempDir(
-      (tmpDir) => {
-        const messageFileName = path.join(tmpDir.path(), 'messages.json');
-        fs.writeFileSync(messageFileName,
-          '{"simulated:" "json syntax error"');
-        return getDefaultLocalizedName({
+  it('checks locale file for malformed json', () => withTempDir(
+    async (tmpDir) => {
+      const messageFileName = path.join(tmpDir.path(), 'messages.json');
+      fs.writeFileSync(messageFileName, '{"simulated:" "json syntax error"');
+      let expectedError;
+      try {
+        await getDefaultLocalizedName({
           messageFile: messageFileName,
           manifestData: manifestWithoutApps,
-        })
-          .then(makeSureItFails())
-          .catch((error) => {
-            assert.instanceOf(error, UsageError);
-            assert.match(error.message, /Unexpected token '"' at 1:15/);
-            assert.match(error.message, /^Error parsing messages.json/);
-            assert.include(error.message, messageFileName);
-          });
+        });
+      } catch (error) {
+        expectedError = error;
       }
-    );
-  });
+      assert.instanceOf(expectedError, UsageError);
+      assert.match(expectedError && expectedError.message,
+                   /Unexpected token '"' at 1:15/);
+      assert.match(expectedError && expectedError.message,
+                   /^Error parsing messages.json/);
+      assert.include(expectedError && expectedError.message, messageFileName);
+    }
+  ));
 
-  it('checks locale file for incorrect format', () => {
-    return withTempDir(
-      (tmpDir) => {
-        const messageFileName = path.join(tmpDir.path(), 'messages.json');
-        //This is missing the 'message' key
-        fs.writeFileSync(messageFileName,
-          `{"extensionName": {
-              "description": "example extension"
-              }
-          }`);
-        const basicLocalizedManifest = {
-          name: '__MSG_extensionName__',
-          version: '0.0.1',
-        };
-        return getDefaultLocalizedName({
+  it('checks locale file for incorrect format', () => withTempDir(
+    async (tmpDir) => {
+      const messageFileName = path.join(tmpDir.path(), 'messages.json');
+      //This is missing the 'message' key
+      fs.writeFileSync(messageFileName,
+        `{"extensionName": {
+            "description": "example extension"
+            }
+        }`);
+      const basicLocalizedManifest = {
+        name: '__MSG_extensionName__',
+        version: '0.0.1',
+      };
+      let expectedError;
+      try {
+        await getDefaultLocalizedName({
           messageFile: messageFileName,
           manifestData: basicLocalizedManifest,
-        })
-          .then(makeSureItFails())
```

Comment:
Same as the above `try { ... } catch { ... }` and the `makeSureItFails` test helper.